src/tools/gophish_import.py

```diff
@@ -281,12 +282,12 @@ def main() -> None:
         with open(args["ASSESSMENT_FILE"]) as json_file:
             assessment = json.load(json_file)
     except FileNotFoundError as e:
-        logging.error(f"{e}\n")
+        logging.error("%s", e)
```

Comment:
Why not use [`logging.exception()`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) here instead of recreating the wheel?

Reply:
Yep, ```logging.exception()``` would be better off here, so its been changed 

See 44e24edb9ac96eb42759bdea8a9229563811ca59

Reply:
Updated again here 3ee423033fefa9873fd13190c0e6edb575cd0080